src/modules/shared/pages/innovations/innovations-advanced-review.component.html

```diff
@@ -1,9 +1,13 @@
 <theme-content-wrapper [status]="pageStatus">
   <div class="nhsuk-grid-row">
-    <div class="nhsuk-grid-column-one-quarter">
+    <div class="nhsuk-grid-column-one-half">
+      <ng-container *ngIf="isAdminType">
+        <h2 class="nhsuk-heading-xs nhsuk-u-margin-0">Search innovation by name or email</h2>
+      </ng-container>
+      <ng-container *ngIf="isAccessorType">
+        <h2 class="nhsuk-heading-xs nhsuk-u-margin-0">Search innovation by name</h2>
+      </ng-container>
```

Comment:
Since each container only has one element, I would do the `*ngIf` directly, but if you think it is more readable this way, leave it as is!

Reply:
It reads well your way as well, will change it :)